public/data/firestore-data.json

```diff
@@ -2169,13 +2169,28 @@
             "title": "The Keynote",
             "videoId": "gpwLAaAVcLA"
           }
+        ],
+        "2019": [
+          {
+            "presentation": "https://www.slideshare.net/utpalbetai/gdg-dev-fest-2019",
+            "tags": [
+              "The Keynote"
+            ],
+            "title": "Keynote - Future is here",
+            "videoId": "zVXCQW1cP20"
+          }
         ]
       },
       "socials": [
         {
           "icon": "twitter",
           "link": "https://twitter.com/utpalbetai",
           "name": "Twitter"
+        },
+        {
+          "icon": "facebook",
```

Comment:
Remove facebook profile from socials

Reply:
I will make this changes in next PR on behalf of you.

Reply:
Thanks @JaldeepAsodariya :)